metadata.yaml

```diff
@@ -1,5 +1,9 @@
 homepage: "https://ogury.com"
 documentation: "https://learn.ogury.com/ogury-pixel-integration-guide/"
 versions:
+  #Latest version
+  - sha: adead46e6d26f3330aaa48a78af6361d7c564aba
+    changeNotes: Adding src as query param
```

Comment:
`Adding source of the post click events
`
Publisher will see this changelog, so maybe it's better to write something like that?